src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ReferencesPage/ImplicitUsingsValueProvider.cs

```diff
@@ -0,0 +1,200 @@
+// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[ExportInterceptingPropertyValueProvider("ImplicitUsingsEditor", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+[AppliesTo(ProjectCapability.CSharp)]
+internal class ImplicitUsingsValueProvider : InterceptingPropertyValueProviderBase
+{
+    private readonly IProjectAccessor _projectAccessor;
+    private readonly ConfiguredProject _configuredProject;
+    private readonly IProjectThreadingService _threadingService;
+
+    [ImportingConstructor]
+    public ImplicitUsingsValueProvider(IProjectAccessor projectAccessor, ConfiguredProject configuredProject, IProjectThreadingService threadingService)
+    {
+        _projectAccessor = projectAccessor;
+        _configuredProject = configuredProject;
+        _threadingService = threadingService;
+    }
+
+    private async Task<List<ImplicitUsing>> GetImplicitUsingsAsync()
+    {
+        List<ImplicitUsing> usings = await _projectAccessor.OpenProjectForReadAsync(_configuredProject, project =>
+        {
+            return project
+                .GetItems("Using")
+                .Select(item =>
+                {
+                    string? isStaticMetadata = item.DirectMetadata.FirstOrDefault(metadata => metadata.Name.Equals("Static", StringComparison.Ordinal))?.EvaluatedValue;
```

Comment:
Prefer using one of the values in `StringComparisons` over `StringComparison.Ordinal`/`StringComparison.OrdinalIgnoreCase`/etc.